ivy/functional/ivy/activations.py

```diff
@@ -376,8 +376,13 @@ def relu(
 @to_native_arrays_and_back
 @handle_array_function
 @handle_device_shifting
+@handle_complex_input
 def sigmoid(
```

Comment:
Complex numbers are marked as unsupported for `sigmoid` in the `tf` and `paddle` backends

Reply:
To fix this, I have to add a jax-like function for sigmoid. I just did in the latest commit

Reply:
Are you sure that's needed? I just tested on the main branch and `ivy.sigmoid(3+4j)` works just fine with both paddle and tensorflow backends

Reply:
Running the test without the sigmoid jax-like function gives errors in tensorflow and paddle.

`[cpu-paddle-False-False] Failed: [undefined]AssertionError:  the results from backend paddle and ground truth framework jax do not match
 (0.2689414322376251+0j)!=(0.21795842936625076-0.20194822765801285j) `

`[cpu-tensorflow-False-False] Failed: [undefined]AssertionError:  the results from backend tensorflow and ground truth framework jax do not match
 (1+0j)!=-0j `

Reply:
Do you still have the test cases that returned those errors?

Reply:
For Paddle:
`Falsifying example: test_jax_sigmoid(
    on_device='cpu',
    frontend='jax',
    backend_fw='paddle',
    dtype_and_x=(['complex128'], [array(-1.-1.j)]),
    test_flags=FrontendFunctionTestFlags(
        num_positional_args=0,
        with_out=False,
        inplace=False,
        as_variable=[False],
        native_arrays=[False],
        test_compile=False,
        generate_frontend_arrays=False,
    ),
    fn_tree='ivy.functional.frontends.jax.nn.sigmoid',
)

You can reproduce this example by temporarily adding @reproduce_failure('6.82.5', b'AXicY2QAAkYGCGBEYzMwAAAAfgAG') as a decorator on your test case`

For Tensorflow:
`Falsifying example: test_jax_sigmoid(
    on_device='cpu',
    frontend='jax',
    backend_fw='tensorflow',
    dtype_and_x=(['complex128'], [array(-709.-1.j)]),
    test_flags=FrontendFunctionTestFlags(
        num_positional_args=0,
        with_out=False,
        inplace=False,
        as_variable=[False],
        native_arrays=[False],
        test_compile=False,
        generate_frontend_arrays=False,
    ),
    fn_tree='ivy.functional.frontends.jax.nn.sigmoid',
)

You can reproduce this example by temporarily adding @reproduce_failure('6.82.5', b'AXicY2QAAkYGMGA6imBDaQAObADM') as a decorator on your test case
@handle_frontend_test(`